style.css

```diff
@@ -150,112 +136,102 @@
   text-decoration: none;
   padding-inline-start: 45px;
 }
-
-.header-left__nav li {
+.nav__item {
   text-transform: uppercase;
   list-style-type: none;          
   margin-right: 35px;
   margin-bottom: 15px;
 }
-
-.header a {
+.nav__link, 
+.sign-in__link, 
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/Mandri_Firsttask/pullRequest?prid=5482251)